data/json/items/gun/shot.json

```diff
@@ -42,6 +42,7 @@
     "color": "light_gray",
     "skill": "shotgun",
     "ammo": "shot",
+    "range": 1,
```

Comment:
Wiki says it should have a effective range of ~50 meters, compared to the ~40 meter effective range of the Mossberg 500. Unless somebody has a good reason for the recoil of 120, I think it should be reduced as well, now that the pseudo-burst setting has been removed.